node_modules/gh-series/tests/test-aggregation.js

```diff
@@ -0,0 +1,206 @@
+                                SeriesTestUtil.assertGetSeries(global.tests.admins.cam2014.client, series.id, false, true, null, function(series) {
+                                    assert.deepEqual(series.metadata.locations, ['Room 3']);
+                                    return callback();
+                                });
+                            });
+                        });
+                    });
+                });
+            });
+        });
+
+        /**
+         * Test that verifies that series metadata is updated when events their organisers change
+         */
+        it('verify series metadata is updated when events their organisers change', function(callback) {
+            // Create a series with 3 events that can aggregate
+            setupAggregation(function(eventA, eventB, eventC, series) {
+                // Sanity check the organisers aggregated
+                assert.deepEqual(series.metadata.organisers, ['Dr Jack Daniels', 'Prof William Lawsons']);
+
+                // Add a plain-text organiser to an event
+                EventsTestUtil.assertUpdateEventOrganisers(global.tests.admins.cam2014.client, eventA.id, {'Father Jack McDougal': true}, function() {
+                    // Get the new event metadata and verify the new organiser is now in the metadata
+                    SeriesTestUtil.assertGetSeries(global.tests.admins.cam2014.client, series.id, false, true, null, function(series) {
+                        assert.deepEqual(series.metadata.organisers, ['Dr Jack Daniels', 'Father Jack McDougal', 'Prof William Lawsons']);
+
+                        // All events should have the same organiser
+                        EventsTestUtil.assertUpdateEventOrganisers(global.tests.admins.cam2014.client, eventA.id, {'Father Jack McDougal': true, 'Dr Jack Daniels': false, 'Prof William Lawsons': false}, function() {
+                            EventsTestUtil.assertUpdateEventOrganisers(global.tests.admins.cam2014.client, eventB.id, {'Father Jack McDougal': true, 'Dr Jack Daniels': false, 'Prof William Lawsons': false}, function() {
+                                EventsTestUtil.assertUpdateEventOrganisers(global.tests.admins.cam2014.client, eventC.id, {'Father Jack McDougal': true, 'Dr Jack Daniels': false, 'Prof William Lawsons': false}, function() {
+                                    SeriesTestUtil.assertGetSeries(global.tests.admins.cam2014.client, series.id, false, true, null, function(series) {
+                                        assert.deepEqual(series.metadata.organisers, ['Father Jack McDougal']);
+
+                                        // Add a user as an organiser
+                                        TestsUtil.generateTestUsers(global.tests.apps.cam2014, 1, false, function(simon) {
+                                            var update = {};
+                                            update[simon.profile.id] = true;
```

Comment:
FWIW, we can now do `_.ghObj(simon.profile.id, true);` to build this update object :)